src/ResourceManager/Resources/Commands.ResourceManager/Cmdlets/Entities/Policy/PolicyDefinitionMode.cs

```diff
@@ -27,6 +27,11 @@ public enum PolicyDefinitionMode
         /// <summary>
         /// The policy definition mode is all.
         /// </summary>
-        All
+        All,
+
+        /// <summary>
+        /// The policy definition mode is not known.
+        /// </summary>
+        NotSpecified
```

Comment:
I'd vote ```None``` as a better name for this.

Reply:
NotSpecified is used in the .NET SDK.  Trying to keep these consistent